uniffi_bindgen/src/scaffolding/templates/scaffolding_template.rs

```diff
@@ -2,6 +2,13 @@
 // Trust me, you don't want to mess with it!
 {% import "macros.rs" as rs %}
 
+// Unit struct that handles lowering/lifting/serialization.  We define `uniffi::FfiConverter<T>` on
+// it for each T that we want to pass across the FFI
+//
+// This is pub, since we need to access it to support external types
+#[doc(hidden)]
+pub struct UniFFIConverter;
```

Comment:
If we accept this PR, then we're committing to always generating a struct like this.  This means that once we support crates using proc-macros only and not UDL files, users will need to call a macro in the crate root to generate this.

I think this is fine and there's a good chance we will want to use it for other things.  It could also generate the `uniffi_types` module for example.